### PR TITLE
Bugfix/bugfix 47

### PR DESCRIPTION
**Pull request by Diogo12246**
fixed the calendar bug by refactoring some code and implementing moment js

### CODE REVIEW

File: demand-capacity-mgmt-frontend/src/components/WeeklyView.tsx [hunk 1]
```diff
@@ -26,73 +26,95 @@ import { DemandCategoryContext } from '../contexts/DemandCategoryProvider';
 import { Demand, DemandCategory, DemandProp, DemandSeriesValue, MaterialDemandSery } from '../interfaces/demand_interfaces';
 import { Button, ButtonGroup, ToggleButton, OverlayTrigger, Tooltip } from 'react-bootstrap';
 import { DemandContext } from '../contexts/DemandContextProvider';
+import moment from 'moment';
```
Comment (Ruskyy) [inline demand-capacity-mgmt-frontend/src/components/WeeklyView.tsx:29]: Missing updated Dependencies File

**Review by Ruskyy — COMMENTED** [review]
Please fix the following issues

**Comment by Bailonis** [comment]
@Ruskyy I think it is all good now

**Comment by Ruskyy** [comment]
Yes, its OK now!

**Comment by carslen** [comment]
@Bailonis please resolve merge conflicts

**Comment by Bailonis** [comment]
@carslen resolved

**Comment by carslen** [comment]
There is still, or after merging #18 again a merge conflict. This is a good example, to not fix the same problem in different branches. @Bailonis please rebase again and resolve.

**Review by carslen — APPROVED** [review]
LGTM

**Pull request merged**